triagebot.toml

```diff
@@ -1057,3 +1057,7 @@ project-exploit-mitigations = [
 # Enable tracking of PR review assignment
 # Documentation at: https://forge.rust-lang.org/triagebot/pr-assignment-tracking.html
 [pr-tracking]
+
+# Enable issue transfers within the org
+# Documentation at: https://forge.rust-lang.org/triagebot/transfer.html
```

Comment:
Location doesn't exist yet, depends on rust-lang/rust-forge#741. The URL is unlikely to change I'd say, so not blocking imo.